lib/engine/game/g_1822_pnw/game.rb

```diff
@@ -391,7 +442,7 @@ def setup_companies
 
           minors = @companies.select { |c| c.id[0] == self.class::COMPANY_MINOR_PREFIX }
           minor_6, minors = minors.partition { |c| c.id == 'M6' }
-          minors_assoc, minors = minors.partition { |c| MINOR_ASSOCIATIONS.key?(c.id) }
+          minors_assoc, minors = minors.partition { |c| MINOR_ASSOCIATIONS.key?(c.id[1..-1]) }
```

Comment:
What's this for?

Reply:
[1..-1] takes the first character off the of the string of the id (which is always 'M') so that it can match up against the IDs in MINOR_ASSOCIATIONS.  Most of the places this is checked it's being done against the corporation ID (which does not contain the "M") instead of against the company ID (which does).

Reply:
Okay. A well-named helper method will make the code more readable for others.

Reply:
Ok.  Made helper method.